src/macrofier/engine.mjs

```diff
@@ -158,7 +158,7 @@ const getComponentExternalSchema = (json) => {
         let title = ''
         if (ref.includes('x-schemas')) {
           if (ref.split('/')[2] !== json.info.title) {
-            title = ref.split('/')[2]
+            title = ref.split('/')[2].toLowerCase()
```

Comment:
It's hypothetically possible to have two unique schemas whose only different in title is case, e.g. `timezone` and `timeZone`.

We should never be doing this in the core codebase, aside from supporting a macro to output it into a template.

Reply:
will revert, since this case conversion is already handling using info.title.lowercase

Reply:
done